tests/charts/psql-service/0.1.8/src/templates/deployment.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.k8Name }}
+  #namespace: {{ .Values.k8Project }}
```

Comment:
```suggestion
```

Reply:
For the changes in the chart - I just copied the chart from development repo, so I don't see any value in updating here until we have a common place to store charts.